src/internal/m365/onedrive/restore_test.go

```diff
@@ -315,3 +320,74 @@ func (suite *RestoreUnitSuite) TestAugmentRestorePaths_DifferentRestorePath() {
 		})
 	}
 }
+
+func (suite *RestoreUnitSuite) TestRestoreItem_errItemAlreadyExists() {
+	table := []struct {
+		name          string
+		onCollision   control.CollisionPolicy
+		expectErr     func(*testing.T, error)
+		expectSkipped assert.BoolAssertionFunc
+	}{
+		{
+			name:        "skip",
+			onCollision: control.Skip,
+			expectErr: func(t *testing.T, err error) {
+				require.NoError(t, err, clues.ToCore(err))
+			},
+			expectSkipped: assert.True,
+		},
+		{
+			name:        "replace",
```

Comment:
should this be "fail"? Ideally "replace" will eventually not return an error (assuming we workaround graph's current weirdness)

Reply:
The name was chosen to describe what conflictBehavior the graph api call is using, based on the onCollision value.  Should be appropriate to keep it as is.  Also, yes, ideally we'll not error, whether or not it's fixed by graph.